hack/docgen/main.go

```diff
@@ -346,6 +348,8 @@ func collectFields(s *structType, aliases map[string]aliasType) (fields []*Field
 	for _, f := range s.node.Fields.List {
 		if f.Names == nil {
 			// This is an embedded struct.
+			fields = append(fields, &Field{Type: "unknown"})
```

Comment:
How will it look like in the docs?
Or that should never be rendered?

Reply:
it is like below in the same block, it gets skipped in the docs

the problem was that with the embedded struct the field offset gets off by one, so the comment blocks are attached to the wrong fields when marshalling

all new documents look like that:

```go
struct SomeConfig{
   meta.Meta

   SomeValue string
}```

So without it, the index of the fields is off by one, but with this fix it is correct. the `meta.Meta` is not documented either way

Reply:
ah, I see